### PR TITLE
Tets pull  

### PR DESCRIPTION
**Pull request by Modificks**

### CODE REVIEW

File: src/main/java/com/lessons/home/springsecurity/configurations/CorsConfiguration.java [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+
```
Comment (vitalik5212) [inline src/main/java/com/lessons/home/springsecurity/configurations/CorsConfiguration.java:5]: What is that?

Comment (Modificks) [reply src/main/java/com/lessons/home/springsecurity/configurations/CorsConfiguration.java:5]: Java Code

Comment (vitalik5212) [reply src/main/java/com/lessons/home/springsecurity/configurations/CorsConfiguration.java:5]: This is the bad change

**Review by vitalik5212 — CHANGES_REQUESTED** [review]

File: src/main/java/com/lessons/home/springsecurity/configurations/CorsConfiguration.java (hunk 1 above)
Comment (Modificks) [reply src/main/java/com/lessons/home/springsecurity/configurations/CorsConfiguration.java:5]: Chort

**Pull request closed without merging**